src/test/java/org/jenkinsci/plugins/badge/ParameterResolverTest.java

```diff
@@ -1,14 +1,16 @@
 package org.jenkinsci.plugins.badge;
 
 import static org.hamcrest.CoreMatchers.is;
+import static org.hamcrest.CoreMatchers.nullValue;
 import static org.hamcrest.MatcherAssert.assertThat;
 
 import hudson.model.Actionable;
+import org.junit.Test;
```

Comment:
Thanks very much for the pull request.  This is a test class that uses JUnit 5 to run the test, but you've used imported for a JUnit 4 test.  I think that you want a JUnit 5 Test annotation rather than a JUnit 4 Test annotation.
```suggestion
import org.junit.jupiter.api.Test
```

Reply:
added JUnit 5 instead of JUnit 4